src/deepsparse/server/config.py

```diff
@@ -164,8 +165,8 @@ class ServerConfig(BaseModel):
     )
 
     integration: str = Field(
-        default=INTEGRATION_LOCAL,
-        description="The kind of integration to use. local|sagemaker",
+        default=None,
+        description="The kind of integration to use. local|sagemaker|openai",
```

Comment:
nit: could directly put INTEGRATIONS in the description
```suggestion
        description=f"The kind of integration to use. {INTEGRATIONS}",
```